app/views/themes/reshare_show/hyrax/base/_metadata.html.erb

```diff
@@ -0,0 +1,5 @@
+<dl class="work-show <%= dom_class(presenter) %> mb-0" <%= presenter.microdata_type_to_html %>>
+  <%= render 'attribute_rows', presenter: presenter %>
```

Comment:
If you can use the base attribute rows, you can change this to add the path to it. `render '../../hyrax/base/attribute_rows'` or something like that

Reply:
unfortunately we can't use the base/attribute_rows because of the render as:faceted thing